packages/validators/package.json

```diff
@@ -39,9 +56,5 @@
   ],
   "publishConfig": {
     "access": "public"
-  },
-  "files": [
-    "dist",
-    "index.d.ts"
-  ]
```

Comment:
Removing this will cause all files in the repo to published. Is removing it necessary for the fix?

Reply:
Ah I just moved "files" above "exports" to make it consistent with the other `package.json` file. Completely unnecessary change on my part, happy to move it back!